DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,25 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
-        if (!isset($config['default_formatter'])) {
-            reset($config['formatters']);
-            $config['default_formatter'] = key($config['formatters']);
-        }
-
         if (!array_key_exists($config['default_formatter'], $config['formatters'])) {
-            throw new \InvalidArgumentException(sprintf('SonataFormatterBundle - Invalid default formatter : %s, available : %s', $config['default_formatter'], json_encode(array_keys($config['formatters']))));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
As said, I saw quite a few rounds of debates about this and I am not up to date with the latest decisions on this front, I just wanted to point a possible discrepancy.